java/src/main/java/ai/rapids/cudf/Cuda.java

```diff
@@ -386,6 +386,42 @@ static void asyncMemcpy(long dst, long src, long count, CudaMemcpyKind kind) {
    */
   static native int getNativeComputeMode() throws CudaException;
 
+  /**
+   * Gets the major CUDA compute capability of the current device.
+   * 
+   * For reference: https://developer.nvidia.com/cuda-gpus
+   * Hardware Generation	Compute Capability
+   *     Ampere	                8.x
+   *     Turing	                7.5
+   *     Volta	                7.0, 7.2
+   *     Pascal	                6.x
+   *     Maxwell                5.x
+   *     Kepler	                3.x
+   *     Fermi	                2.x
+   * 
+   * @return The Major compute capability version number of the current CUDA device
+   * @throws CudaException on any error
+   */
+  public static native int getCurrentComputeCapabilityMajor() throws CudaException;  
```

Comment:
Nit: Why "Current" in the API names?  We don't use "Current" in `getRuntimeVersion`, `getNativeComputeMode` or other similar query methods.

Reply:
Should the fact that the API works on the current CUDA device be reflected in the name?

Reply:
IMO no, it's implied, e.g.: cudaGetDevice isn't cudaGetCurrentDevice, for example.  But again, it's a minor point and I'm fine if we leave it as-is.

Reply:
> cudaGetDevice

Agreed. Again a matter of overthinking it. This is fixed now.